File: Python/Python Basics/ControlStatement.py
# python - control statement

# el-if ladder
x = int(input("Enter a Number"))
if(x>0)     : {print(x," >> 0")}
elif (x<0)  : {print(x," << 0")}
else        : {print(x," == 0")}

# while loop - Executes till condition is satisfied
print("Table of ",x)
i = 0
while i<=10 : 
    print(i," * ",x," = ",x*i,end=' ')
    i = i+1

# for loop - (iterrator : iterable)

ans = 1
i = 1
print("Factorial of ",x)
for i in range(1,x+1) :
    ans = ans * i
    i = i + 1
print(" is ", ans)

File: PHP/PHP Basics/ControlStatement.php
<!-- PHP - Control Statement -->
<?php

// if-else statement
$bool = true;
if($bool) {
    echo "True value \n";
}
else { 
    echo "False value \n";
}

// for loop
$int = 5;
for ($i=0; $i < $int; $i++) { 
    # code...
    echo ($i * $int),"\n";
}

// while / dp-while loop
do{
    echo "count = ",$int, "\n";
    $int--;
}while($int>0)

?>